app/controllers/debit/collections/show.js

```diff
@@ -21,6 +21,9 @@ export default Controller.extend({
       transactionsGroupedByUser.transactions.pushObject(transaction);
       transactionsGroupedByUser.totalTransactionAmount += transaction.get('amount');
     });
+    transactions.forEach(transaction => {
```

Comment:
I don't think this is the way to go. Firstly we round all the transactions independent, which could cause a difference in the amount that the total says and the real amount we will debit. I think we should do the rounding at the end, so on the total itself. 

Also I do not understand why we multiply by 100 and then devide by 100, could you explain?

Reply:
As for your first comment, We don't round all transactions independently, we round only the totals of every user. 

Every total is multiplied by 100 because this prevents the last 2 decimals from being rounded. Everything that's still a decimal after multiplication by 100 will be rounded, so after division by 100 at most 2 decimals remain. (multiplying by 100 moves the comma by two digits)

Reply:
I just realized that I wrote confusing code.

There's two different arrays here, one is an array of per-user-grouped transactions ('transactions', which is initialized as an empty array), the other is 'this.get('model.transactions'), which is a list of all separate expenses.

In hindsight, I see that I made a confusing decision in the variable naming. in line 24, the variable 'transaction' should be named 'transactionsGroupedByUser', since it's not a separate transaction, but one of the elements pushed to the 'transactions' array.

Reply:
Thanks for the explanation. How about using JavaScript `toFixed`? This actually does this behavior without the multiplication/dividing mechanism.

 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed

Reply:
thanks for the suggestion, this is exactly what I needed.